.github/workflows/performance.yml

```diff
@@ -0,0 +1,208 @@
+name: Performance Tests
+
+on:
+  push:
+    branches:
+      - trunk
+      - '6.[2-9]'
+      - '[7-9].[0-9]'
+    tags:
+      - '[6-9]+.[2-9]'
+      - '[6-9]+.[2-9].[0-9]+'
+  pull_request:
+    branches:
+      - trunk
+      - '6.[2-9]'
+      - '[7-9].[0-9]'
+
+# Cancels all previous workflow runs for pull requests that have not completed.
+concurrency:
+  # The concurrency group contains the workflow name and the branch name for pull requests
+  # or the commit hash for any other events.
+  group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.sha }}
+  cancel-in-progress: true
+
+env:
+  LOCAL_DIR: build
+  BUILD_PATH: '--path=/var/www/build'
```

Comment:
Don't feel strongly about this, but when I suggested this environment variable, I envisioned keeping `--path=` inline for the CLI command, and using it like `--path=${{ env.BUILD_PATH }}`.

Don't think we need the raw path anywhere right now, but could come in handy to have in the future?

Reply:
I wondered the same thing, or even remove the extra environment var and do `--path=/var/www/${{ env.LOCAL_DIR }}`

Reply:
Updated in [2e15306](https://github.com/WordPress/wordpress-develop/pull/4139/commits/2e153061a54d7fbbb46cf7b356a18285d61f2931)

Reply:
Thanks Joe.  I misunderstood it.